modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterState.java

```diff
@@ -64,7 +63,7 @@ default IgniteProductVersion igniteVersion() {
     }
 
     /**
-     * Returns a REST authentication configuration that should be applied.
+     * Returns a cluster configuration that should be applied.
      */
-    Authentication restAuthToApply();
+    String clusterConfigurationToApply();
```

Comment:
Why is this a part of cluster state?
"toApply" is a confusing naming. Who applies it? When? Javadoc doesn't help, I can't understand this code.